README.md

```diff
@@ -32,7 +32,7 @@ From source:
 ```bash
 git clone https://github.com/RedHatQE/openshift-python-wrapper.git
 cd openshift-python-wrapper
-python setup.py install --user
+uv pip install . # Add -e for editable/development mode if making local changes
```

Comment:
I think there is no point install it on the same folder under same venv of uv (uv sync will have the code anyway)
The only option I think is valid is if someone want to develope and test changes in the wrapper with another project, in this case it should be something like:

git clone https://github.com/RedHatQE/openshift-python-wrapper.git
cd openshift-python-wrapper
pwd
and in the other project do uv/pip/.... install ${pwd}.

WDYT?